//
// Juniper P4 Agent
//
/// @file  AfiDM.h
/// @brief Afi Data Model
//
// Created by Sandesh Kumar Sodhi, December 2017
// Copyright (c) [2017] Juniper Networks, Inc. All rights reserved.
//
// All rights reserved.
//
// Notice and Disclaimer: This code is licensed to you under the Apache
// License 2.0 (the "License"). You may not use this code except in compliance
// with the License. This code is not an official Juniper product. You can
// obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
//
// Third-Party Code: This code may depend on other components under separate
// copyright notice and license terms. Your use of the source code for those
// components is subject to the terms and conditions of the respective license
// as noted in the Third-Party source code file.
//

#ifndef SRC_AFI_INCLUDE_AFIDM_H_
#define SRC_AFI_INCLUDE_AFIDM_H_

#include "afi_tree/afi_tree.pb.h"
#include "afi_tree_entry/afi_tree_entry.pb.h"
#include "afi_cap/afi_cap.pb.h"
#include "afi_cap_match/afi_cap_match.pb.h"
#include "afi_cap_action/afi_cap_action.pb.h"
#include "afi_cap_entry/afi_cap_entry.pb.h"
#include "afi_cap_entry_match/afi_cap_entry_match.pb.h"
#include "afi_cap_entry_action/afi_cap_entry_action.pb.h"
#include "afi_encap/afi_encap.pb.h"
#include "afi_encap_entry/afi_encap_entry.pb.h"
#include "afi_tree_encap/afi_tree_encap.pb.h"
#include "afi_tree_encap_entry/afi_tree_encap_entry.pb.h"

namespace AFIHAL
{
}  // namespace AFIHAL

#endif  // SRC_AFI_INCLUDE_AFIDM_H_
